src/ai/backend/agent/agent.py

```diff
@@ -270,7 +270,7 @@ def __init__(
         restarting: bool = False,
     ) -> None:
         self.image_labels = kernel_config["image"]["labels"]
-        self.kspec_version = int(self.image_labels.get("ai.backend.kernelspec", "1"))
+        self.kspec_version = int(self.image_labels.get(LabelName.KERNEL_SPEC.value, "1"))
```

Comment:
We already use LabelName as StrEnum. StrEnum is compatible with `str` even if you don't call value and just use the member value.